packages/snapshot-listener/package.json

```diff
@@ -20,15 +20,14 @@
   "dependencies": {
     "@hicommonwealth/adapters": "workspace:*",
     "@hicommonwealth/core": "workspace:*",
-    "@hicommonwealth/logging": "workspace:*",
     "@hicommonwealth/model": "workspace:*",
     "body-parser": "^1.20.1",
-    "dotenv": "^16.0.3",
     "express": "^4.18.2",
     "moment": "^2.23.0",
     "node-fetch": "2",
     "sequelize": "^6.32.1",
-    "tslog": "^3.3.4"
+    "tslog": "^3.3.4",
```

Comment:
I don't think we need this dependency. I removed it in #7542 but it was added back in #7802.